sormas-e2e-tests/src/test/resources/features/sanity/web/Case.feature

```diff
@@ -399,3 +399,96 @@ Feature: Case end to end tests
     And I navigate to the last created through API Event page via URL
     And I check that number of displayed Event Participants is 1
 
+  @issue=SORDEV-6612 @env_main
+  Scenario: Manually triggered calculation of case classification
```

Comment:
we already have a class with existing similar test:
Feature: Case Classification functionality